tests/scripts/kruize_metrics.py

```diff
@@ -0,0 +1,357 @@
+        outputfile = os.path.join(outputdir, "increase_" + resultsfile)
+        print("RUNNING THE JOB TO COLLECT KRUIZE INCREASE METRICS..")
+        results_map = run_queries("increase")
+    elif queries_type == "total":
+        outputfile = os.path.join(outputdir, "total_" + resultsfile)
+        print("RUNNING THE JOB TO COLLECT KRUIZE TOTAL METRICS..")
+        results_map = run_queries("total")
+    results_map['timestamp'] = timestamp_utc
+    write_header_to_csv(outputfile)
+    with open(outputfile, 'a') as f:
+        writer = csv.DictWriter(f, fieldnames=csv_headers)
+        writer.writerow(results_map)
+
+def schedule_job(queries_type):
+    outputdir = "results"
+    if not os.path.exists(outputdir):
+        os.mkdir(outputdir)
+    numeric_time = int(time_duration[:-1])
+    time_in_seconds = numeric_time * 60
+    if getOneDataPoint == "true":
+        job("increase",outputdir)
+        job("total",outputdir)
+    else:
+        if duration is None:
+            while True:
+                job(queries_type,outputdir)
+                print("Sleep for ",time_in_seconds, " seconds")
+                time.sleep(time_in_seconds)
+        else:
+            now = datetime.utcnow()
+            end = now + timedelta(hours=duration)
+            while now < end:
+                now = datetime.utcnow()
+                job(queries_type,outputdir) 
+                print("Sleep for ",time_in_seconds, " seconds")
+                time.sleep(time_in_seconds) 
+
+def main(argv):
+    global duration
+    global cluster_type
+    global server
+    global clusterResults
+    global time_duration 
+    global queries_map
+    global getOneDataPoint
+    global resultsfile
+    global namespace
+
+    try:
+        opts, args = getopt.getopt(argv,"h:c:s:d:t:q:p:r:")
+    except getopt.GetoptError:
+        print("kruize_metrics.py -c <cluster type> -s <server>")
+        sys.exit(2)
+    for opt, arg in opts:
+        if opt == '-h':
+            print("kruize_metrics.py -c <cluster type> -s <server> -t <time duration for a query in mins:Default:60m> -d <duration the script runs in hours:Default:None> -q <query_type:increase/total.Default:increase> -p <single data point:Default:false>")
+            sys.exit()
+        elif opt == '-c':
+            cluster_type = arg
+        elif opt == '-s':
+            server = arg
+        elif opt == '-d':
+            duration = arg
+        elif opt == '-t':
+            time_duration = arg
+        elif opt == '-q':
+            queries_type = arg
+        elif opt == '-p':
+            getOneDataPoint = arg
+        elif opt == '-r':
+            resultsfile = arg
+            
+    if '-t' not in sys.argv:
+        time_duration = "60m"
+    if '-q' not in sys.argv:
+        queries_type = "increase"
+    if '-d' not in sys.argv:
+        duration = None
+    if '-p' not in sys.argv:
+        getOneDataPoint = "false"
+    if '-r' not in sys.argv:
+        resultsfile = "kruizemetrics.csv"
```

Comment:
Both at the start and completion of the script, the output file should be mentioned